### PR TITLE
General fixes and improvements to CR-10 SE, Ender-3 V3 KE, Ender-3 S1 Plus, & small fix for Ender-3 V3 SE

### LINKED ISSUE #4786: GCODE Error, Creality CR10 SE profile
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.0.0

### Operating System (OS)

Windows

### OS Version

Windows 11

### Additional system information

_No response_

### Printer

Creality CR 10 SE

### How to reproduce

Try to slice a file using the just added CR 10 SE profile

### Actual results

```
Failed to generate gcode for invalid custom G-code.
machine_start_gcode Parsing error at line 7: Variable does not exist
M109 S[material_print_temperature_layer_0] 
       ^
Please check the custom G-code or use the default custom G-code.
```

### Expected results

Slice process concluding successful 

### Project file & Debug log uploads

.

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by cochcoder**
This PR is working towards fixing all the errors/bugs in the profiles that I added to Orca Slicer. 

TODO: 

- [x] Fix incorrect line width (mentioned by [exploitfate](https://github.com/exploitfate) in #4653) 
- [x] Fix incorrect line width for CR-10 SE
- [x] Fix CR-10 SE G-code error (by @bkonosky)
- [x] Add changes to CR-10 SE from @bkonosky in #4966 
- [x] Fix incorrect build plate model sizing #4786 
- [x] Fix the incorrect bed size for the Ender-3 S1 Plus 0.8 nozzle type
- [x] Make build plate model for Ender-3 S1 Plus more accurate 
- [x] Fix low default acceleration in CR-10 SE profile (mentioned [here](https://github.com/SoftFever/OrcaSlicer/issues/4786#issuecomment-2033348614))
- [x] Fix CR-10 SE build area
- [x] minor Ender-3 V3 KE improvements (as mentioned by [@Orel-A](https://github.com/SoftFever/OrcaSlicer/pull/4675#issuecomment-2048122901))

Closes #4786

### CODE REVIEW

**Comment by bistory** [comment]
Yep, never rename an existing file or you will break compatibility ;-)
That's what happened with one of your previous commits.
I know it's tempting to rename files and uniformize names but refrain from doing that. I made the same mistakes ;-)

**Comment by cochcoder** [comment]
I am having this strange issue where even though I changed the start G-Code and line width to be correct, they don't apply when I build the program

**Comment by cochcoder** [comment]
To clarify my changes with Flying Bear, I renamed only the names of the filaments, machines, and processes in the hopes of making it so that the front end looks correct without causing issues with any custom profiles that have been made, at the cost of the backend being even more confusing. It may be better to rip off the band-aid and completely fix everything at the cost of custom profiles bring broken, but should be better for future changes.

**Comment by Orel-A** [comment]
There is a bug when settings 0% on retract amount before wipe. It will treat it like we set 52% amount before wipe.
On BambuStudio the bug isn't present.

**Comment by cochcoder** [comment]
> There is a bug when settings 0% on retract amount before wipe. It will treat it like we set 52% amount before wipe.
> 
> On BambuStudio the bug isn't present.

Is this bug also present in the main build or just in this PR?

**Comment by Orel-A** [comment]
> > There is a bug when settings 0% on retract amount before wipe. It will treat it like we set 52% amount before wipe.
> > On BambuStudio the bug isn't present.
> 
> Is this bug also present in the main build or just in this PR?

Main build and even on Bambu A1 printer:

image

It's even retracting more than it should.

(Using 2.1.0-dev)

Maybe it's also present in 2.0.0

**Comment by Orel-A** [comment]
> > There is a bug when settings 0% on retract amount before wipe. It will treat it like we set 52% amount before wipe.
> > On BambuStudio the bug isn't present.
> 
> Is this bug also present in the main build or just in this PR?

Main build:
image

**Comment by cochcoder** [comment]
> > > There is a bug when settings 0% on retract amount before wipe. It will treat it like we set 52% amount before wipe.
> > > On BambuStudio the bug isn't present.
> > 
> > 
> > Is this bug also present in the main build or just in this PR?
> 
> Main build: image

So it's present in the main build? If so, it's out of my wheelhouse.

**Comment by Orel-A** [comment]
I can't test properly because it's also causing overextrusion and spiderwebs:

Same filament, same settings on BambuStudio comes clean!
So I can't trust my tests with OrcaSlicer.

Lets' revert the changes to X-Y hole compensation and Elephant foot compensation, set them both to zero.
Let the end-user determine these values.

File: resources/profiles/Creality/machine/Creality CR-10 SE 0.2 nozzle.json [hunk 1]
```diff
@@ -15,11 +15,11 @@
     ],
     "printable_area": [
       "0x0",
-      "250x0",
-      "250x250",
-      "0x250"
+      "220x0",
+      "220x220",
+      "0x220"
```
Comment (henrivdr) [inline resources/profiles/Creality/machine/Creality CR-10 SE 0.2 nozzle.json:20]: I measured the bed size as 235x235. So 230x230 should be a good value. If the user accounts for the purge line.

File: resources/profiles/Creality/machine/Creality Ender-3 V3 KE 0.4 nozzle.json [hunk 2]
```diff
@@ -119,7 +119,7 @@
...
+	"machine_start_gcode": "M220 S100 ;Reset Feedrate\nM221 S100 ;Reset Flowrate\n\nM140 S[bed_temperature_initial_layer_single] ;Set bed temp\nG28 X Y ;Home XY axes\nM190 S[bed_temperature_initial_layer_single] ;Wait for bed temp to stabilize\nM104 S[nozzle_temperature_initial_layer] ;Set nozzle temp\nG28 Z ;Home Z axis & load bed mesh\n\nG92 E0 ;Reset Extruder\nG1 X-2.0 Y20 Z0.28 F5000.0 ;Move to start position\nM109 S[nozzle_temperature_initial_layer] ;Wait for nozzle temp to stabilize\nG1 X-2.0 Y145.0 Z0.2 F1500.0 E15 ;Draw the first line\nG1 X-1.7 Y145.0 Z0.2 F5000.0 ;Move to side a little\nG1 X-1.7 Y30 Z0.2 F1500.0 E15 ;Draw the second line\nG92 E0 ;Reset Extruder",
```
Comment (henrivdr) [inline resources/profiles/Creality/machine/Creality Ender-3 V3 KE 0.4 nozzle.json:122]: I don't see the "load mesh"

File: resources/profiles/Creality/process/0.12mm Fine @Creality Ender3V3SE 0.2.json [hunk 3]
```diff
@@ -25,20 +25,20 @@
 	"draft_shield": "disabled",
 	"elefant_foot_compensation": "0",
 	"enable_arc_fitting": "0",
-	"outer_wall_line_width": "0.42",
```
Comment (henrivdr) [inline resources/profiles/Creality/process/0.12mm Fine @Creality Ender3V3SE 0.2.json:28]: What is the purpose of this drastic change in line width?

**Review by henrivdr — COMMENTED** [review]
See my comments

File: resources/profiles/Creality/machine/Creality CR-10 SE 0.2 nozzle.json (hunk 1 above)
Comment (bkonosky) [reply resources/profiles/Creality/machine/Creality CR-10 SE 0.2 nozzle.json:20]: In Creality's slicer it says the bed size is 220x220. It's also what is reported in the docs.

Printer.cfg says y goes from -5.5 to 225 and x is -10.5 to 243.

Comment (henrivdr) [reply resources/profiles/Creality/machine/Creality CR-10 SE 0.2 nozzle.json:20]: Ok Creality seems to play it safe also. I see they reduced X max by 2 mm in printer.cfg.
I will resolve this comment.

File: resources/profiles/Creality/machine/Creality Ender-3 V3 KE 0.4 nozzle.json (hunk 2 above)
Comment (Orel-A) [reply resources/profiles/Creality/machine/Creality Ender-3 V3 KE 0.4 nozzle.json:122]: The G28 gcode command loads it. Creality Ender-3 V3 KE comes with a homing override config file that adds it

**Comment by Orel-A** [comment]
> So it's present in the main build? If so, it's out of my wheelhouse.

The problem is caused by this pull request: https://github.com/SoftFever/OrcaSlicer/pull/3888

File: resources/profiles/Creality/machine/Creality Ender-3 V3 KE 0.4 nozzle.json (hunk 2 above)
Comment (cochcoder) [reply resources/profiles/Creality/machine/Creality Ender-3 V3 KE 0.4 nozzle.json]: To be sure, nothing needs to be changed?

**Comment by cochcoder** [comment]
> I can't test properly because it's also causing overextrusion and spiderwebs:  Same filament, same settings on BambuStudio comes clean! So I can't trust my tests with OrcaSlicer.
> 
> Lets' revert the changes to X-Y hole compensation and Elephant foot compensation, set them both to zero. Let the end-user determine these values.

Done, let me know if it fixes the issues

**Comment by Orel-A** [comment]
We'll heat the nozzle after the G28 Z command since it can take some time if a previous print was tall and it's cause oozing on the middle of the bed.
```
M220 S100 ;Reset Feedrate
M221 S100 ;Reset Flowrate

M140 S[bed_temperature_initial_layer_single] ;Set bed temp
G28 X Y ;Home XY axes
M190 S[bed_temperature_initial_layer_single] ;Wait for bed temp to stabilize
G28 Z ;Home Z axis & load bed mesh

M104 S[nozzle_temperature_initial_layer] ;Set nozzle temp
G92 E0 ;Reset Extruder
G1 X-2.0 Y20 Z0.3 F5000.0 ;Move to start position
M109 S[nozzle_temperature_initial_layer] ;Wait for nozzle temp to stabilize
G1 Z0.2 ;Lower nozzle to printing height
G1 Y145.0 F1500.0 E15 ;Draw the first line
G1 X-1.7 F5000.0 ;Move to side a little
G1 Y30 F1500.0 E15 ;Draw the second line
G92 E0 ;Reset Extruder
```

In JSON:
`"machine_start_gcode": "M220 S100 ;Reset Feedrate\nM221 S100 ;Reset Flowrate\n\nM140 S[bed_temperature_initial_layer_single] ;Set bed temp\nG28 X Y ;Home XY axes\nM190 S[bed_temperature_initial_layer_single] ;Wait for bed temp to stabilize\nG28 Z ;Home Z axis & load bed mesh\n\nM104 S[nozzle_temperature_initial_layer] ;Set nozzle temp\nG92 E0 ;Reset Extruder\nG1 X-2.0 Y20 Z0.3 F5000.0 ;Move to start position\nM109 S[nozzle_temperature_initial_layer] ;Wait for nozzle temp to stabilize\nG1 Z0.2 ;Lower nozzle to printing height\nG1 Y145.0 F1500.0 E15 ;Draw the first line\nG1 X-1.7 F5000.0 ;Move to side a little\nG1 Y30 F1500.0 E15 ;Draw the second line\nG92 E0 ;Reset Extruder"`

Regarding the spiderwebs issue, this is a problem with Orca. In BambuStudio after the wiping procedure there is a tiny retraction of around 0.025mm and it's not a bug as some think, that is because during the wiping procedure a thin film of filament is stuck to the nozzle, this small retraction eliminates that.

Anyways, in the meantime, and the small amount of time that I've had for testing; here are some settings that reduce these spiderwebs:
We'll set retraction before wipe to 100%: `"retract_before_wipe": ["100%"]`
And we'll disable role based wipe speed: `"role_based_wipe_speed": "0"` so it will use the 80% speed of the travel speed.

I've also tested 60mm/s retraction speed with zero retract before wipe and a wiping speed of 240mm/s (So that Orca's algorithm will let the whole retraction happen in the wiping procedure) and I couldn't determine which settings are better.

**Comment by cochcoder** [comment]
It seems that adding "role_based_wipe_speed": "0", causes the build to fail. Can you try without this variable enabled, just in case? 

> I've also tested 60mm/s retraction speed with zero retract before wipe and a wiping speed of 240mm/s (So that Orca's algorithm will let the whole retraction happen in the wiping procedure) and I couldn't determine which settings are better.

Adding wipe_speed also causes the build to fail, I am open to having a middle ground between 30 and 60 mm/s for retraction speed to prevent a possible scenario of a filament or printer not liking 60mm/s over a long time (I want to keep these settings conservative and more "stock").

**Comment by Orel-A** [comment]
You put the variable role_based_wipe_speed in the wrong file. It should be in the process files of the ke. And it's not an array.

**Comment by cochcoder** [comment]
Ok, it's in there now

**Comment by Orel-A** [comment]
0.2mm file?

**Comment by cochcoder** [comment]
> 0.2mm file?

Missed that, sorry

**Comment by cochcoder** [comment]
Are there any other changes needed to the Ender-3 V3 KE or is it all good?

**Comment by Orel-A** [comment]
> Ok, it's in there now. I also upped the retraction speed a little, should the wipe speed also be brought up a little?

For retraction speed; it's recommended PLA 45mm/s, PETG 35mm/s and TPU 25mm/s. So I guess leave it as default at 30mm/s.
As for the wipe speed, currently it should be 80% of the travel speed which calculates to 320mm/s. Faster than that and the nozzle will tear off the prints from the bed.

**Comment by cochcoder** [comment]
> As for the wipe speed, currently it should be 80% of the travel speed which calculates to 320mm/s. Faster than that and the nozzle will tear off the prints from the bed.

I am a little concerned that this might be too much for delicate filaments, what is your experience with this?

**Comment by Orel-A** [comment]
> I am a little concerned that this might be too much for delicate filaments, what is your experience with this?

My experience is that it could become a problem with over-extrusion and tall prints. For example knocking tree supports.

**Comment by cochcoder** [comment]
> > I am a little concerned that this might be too much for delicate filaments, what is your experience with this?
> 
> My experience is that it could become a problem with over-extrusion and tall prints. For example knocking tree supports.

Do you have a speed that wouldn't cause issues with tall prints?

**Comment by Orel-A** [comment]
I haven't tested for a definite answer regarding this. However settings the wiping speed to 200mm/s should be safer but over-extrusion will always create issues regardless.

**Comment by cochcoder** [comment]
> I haven't tested for a definite answer regarding this. However settings the wiping speed to 200mm/s should be safer but over-extrusion will always create issues regardless.

I'll implement this, I just want to make sure that new users won't have any issues when printing tall narrow objects.

**Comment by cochcoder** [comment]
If there is nothing else you can think of that's needed. I'll go ahead and mark this PR as ready for review.

**Comment by Orel-A** [comment]
you missed the 0.16mm profile.

if there was an option for a button in Orca to click "print & calibrate" it would have been good to add BED_MESH_CALIBRATE after the G28 gcode to have that same effect. same as in Creality Print. would have been good.

**Comment by cochcoder** [comment]
Should I try to add `BED_MESH_CALIBRATE` after the bed temperature reaches temp?

**Comment by cochcoder** [comment]
> you missed the 0.16mm profile.

Thanks for catching that one!

**Comment by Orel-A** [comment]
> Should I try to add `BED_MESH_CALIBRATE` after the bed temperature reaches temp?

The place for it is right after the `G28 Z`.
I'll test tomorrow if `BED_MESH_CALIBRATE PROBE_COUNT=6,6` works (the default is 5 by 5 in printer.cfg).

**Comment by Orel-A** [comment]
It's working, can you make the changes?

**Comment by cochcoder** [comment]
Please check this [build](https://github.com/cochcoder/OrcaSlicer/actions/runs/8908559424) when you can to make sure that everything is working as intended on your side.

**Comment by Orel-A** [comment]
working good, perfect first layer

**Comment by cochcoder** [comment]
Thanks for your help!

**Comment by SoftFever** [comment]
Hey guys,
What's the status of this PR?

@cochcoder could you give this PR a more descriptive title?

**Comment by cochcoder** [comment]
> Hey guys, What's the status of this PR?
> 
> @cochcoder could you give this PR a more descriptive title?

I think this PR is ready to merge! I'll give a more descriptive title...

**Comment by Orel-A** [comment]
I noticed a problem when printing on PolyMaker PolyLite PETG.
On 0.2mm layer height the Infill Combination prints the infill with 0.4mm layer height, and it's not sticking to the previous layer.
I didn't have this problem with High Speed PETG. This can be mitigated somewhat with 255C nozzle temperature.
Futhermore the V3 KE has a minimum fan speed of 55%:
image
`variable_fan0_min` is set to 140 in the config as default.
That means that the 40% fan speed on PETG profile is converted to 74%...

**Comment by Orel-A** [comment]
I noticed that on Matte PLA there is a bit of a change in color due to the high speed printing. Let's reduce the speed to 200mm/s on outer wall.
We'll increase Internal Solid Infill to 300mm/s and increase its line width to 0.45mm.
Acceleration to deceleration should be disabled in the slicer `"accel_to_decel_enable": "0"`, this will keep the `max_accel_to_decel: 2500` in printer.cfg instead of the slicer, making it static.
Regarding the Infill Combination and what I said in the previous post, I'm not sure if we should disable it since I only had this issue with PolyMaker PETG... What are you thoughts on this? Also, this might be because how to config sets the cooling fans speeds.
However, on the other hand, stacking layers on infills can make weak prints due to weak layer adhesion.

**Comment by cochcoder** [comment]
I want to try and see if the fix for the fan will work before we disable the infill combination

**Comment by cochcoder** [comment]
> EDIT: I've found a fix for the fan speed, we can add a change to the variable:
SET_GCODE_VARIABLE MACRO=PRINTER_PARAM VARIABLE=fan0_min VALUE=30
The value is 30 since it's when the fans start to spin.

Would this go in start g-code?

**Comment by Orel-A** [comment]
> Would this go in start g-code?

Yes.

I'm testing the Infill Combination with the fan speed fix.

**Comment by Orel-A** [comment]
When using CANCEL_PRINT there is a bug, it's changing the accel_to_decel value to 5000mm/s:
image
On the next print it won't return it.
We need to add `SET_VELOCITY_LIMIT ACCEL_TO_DECEL=2500` to overwrite what FINISH_INIT macro did.

**Comment by Orel-A** [comment]
It should not be in machine end gcode because it will never be called since you cancel the print and it won't reach the machine end gcode (The bug happens when canceling a print since CANCEL_PRINT macro calls the FINISH_INIT macro).
Need to place it in machine start gcode.

**Comment by Orel-A** [comment]
Okay regarding the the Infill Combination; the fan fix + increasing the nozzle temperature to 265C fixed the problem for me, so we'll leave it enabled.

**Comment by cochcoder** [comment]
> Okay regarding the the Infill Combination; the fan fix + increasing the nozzle temperature to 265C fixed the problem for me, so we'll leave it enabled.

What if you don't increase the nozzle temp?

**Comment by cochcoder** [comment]
How are the other profiles? (fine, draft, etc.)

**Comment by Orel-A** [comment]
> How are the other profiles? (fine, draft, etc.)

I didn't test. Don't have much time for it. I report issues when I encounter them. But we've done a great job thus far.
I won't use this filament in the future, can't print at high speeds. I have better purchase options at my local store.
Layer adhesion defintely improved thanks to the fan speed fix.

**Comment by cochcoder** [comment]
Would you say that the changes we've made work in all of the profiles?

**Comment by Orel-A** [comment]
All the changes we've made are improving on the existing profiles, they're not as well tuned as BambuLab's, but nonetheless do a good job.
But I can see we missed some things, for example:
1. For supports settings, the top z distance and bottom z distance shouldn't be higher than the layer height.
2. The threshold angle for supports isn't well tuned.

**For layer height 0.12mm:**
Decrease top z distance and bottom z distance to 0.12mm to match layer height
Decrease supports threshold angle to 20 degrees
Change top interface layers to 2

**For layer height 0.16mm:**
Decrease bottom z distance to 0.16mm to match layer height
Decrease supports threshold angle to 25 degrees
Change top interface layers to 2

**For layer height 0.24mm:**
Increase supports threshold angle to 35 degrees

**Comment by cochcoder** [comment]
I'm not sure about decreasing the support's distance from the print to one layer as that may only have negative effects. What's your experience with this?

**Comment by Orel-A** [comment]
Done some tests. Nothing conclusive.

With a top Z distance of 0.12mm when you pull the supports the Support Interface is more likely to get stuck to the print but after clean up there isn't too much scarring. But that depends on the filament and what temperature you are printing, so if you have a well tuned settings for overhangs it will perform well with regards to scarring.
So I guess we will leave it at 0.15mm or 0.16mm.
I tested on 0.16mm layer height with 0.16mm Z distance, and it performed well.

For 0.24mm layer height in the settings you enabled Infill Combination that does nothing since two times the size of the layer is well above the 0.4mm nozzle size.
I think we should disable Infill Combination for 0.2mm layer height and leave it enabled for 0.16mm and 0.12mm. I tested on PolyMaker PolyTerra PLA+ this time and I think it's causing issues with weak layer adhestion.

**Comment by cochcoder** [comment]
> EDIT: On 0.12mm layer height it's using 0.36mm infill height, I thought it would only use double its size, meaning 0.24mm. And on 0.16mm it's indeed 0.32mm. Hmm... Is it just my filament? I'm printing at 210C temperature. The filament has a die swell effect in contrast to eSun PLA+ which I didn't have this phenomenon.

I don't know why this is happening as I don't see anything in the profile's settings that would affect this.

Do you see any other issues or is this PR ready to merge? Take a few days to test if you need to.

**Review by Orel-A — APPROVED** [review]
Yes, ready to merge. Good fixes for V3 KE needed to be merged here.

**Comment by cochcoder** [comment]
@SoftFever this PR is ready to be merged

**Comment by Orel-A** [comment]
Hi, can you increase the Tree Support Branch Diameter from 2mm to 5mm? 2mm is too small to have a stable foothold on the bed.

**Comment by cochcoder** [comment]
> @cochcoder In machine end gcode, please change the line: `G1 Z20 ;Raise Z more` To: `G1 Z{if max_layer_z < 50}25{else}5{endif} ;Raise Z more`
> 
> I got an error today, this will prevent the error. Max permittable Z height is 246mm. So with printable height of 240mm the 5mm Z increase is still safe.

Done!

**Comment by SoftFever** [comment]
@cochcoder 
Thank you very much.
There are some conflicts, could you take a look?

**Comment by cochcoder** [comment]
> @cochcoder Thank you very much. There are some conflicts, could you take a look?

Done!

**Review by SoftFever — APPROVED** [review]
Looks good
Thank you!

**Comment by cochcoder** [comment]
@Orel-A Do you also experience this? (Just checking in case of different firmware versions). Does the same happen if you upload a copy to the printer over wifi?

**Comment by numToStr** [comment]
@cochcoder 

I have the same setup as @lmutt and can confirm the same experience/behaviour as explained above on every print with v2.1.0-beta . Although I usually upload gcode and start printing via the web interface.

I knew something was different from 2.0.0 but being a newbie i didn't know this is an unexpected behaviour.

**Comment by cochcoder** [comment]
@numToStr Thanks for conforming this isn't a single device issue. I'll get on it!

**Comment by Orel-A** [comment]
If you tick the Calibrate checkbox while selecting the file uploaded via Orca on the screen of the printer, of course it'll do calibration twice. Just don't tick the Calibrate button on the printer. I don't see it as an issue. Unless you have a method to detect in the start gcode if the printer was already homed and skip the procedure.

**Comment by cochcoder** [comment]
> If you tick the Calibrate checkbox while selecting the file uploaded via Orca on the screen of the printer, of course it'll do calibration twice. Just don't tick the Calibrate button on the printer. I don't see it as an issue. Unless you have a method to detect in the start gcode if the printer was already homed and skip the procedure.

What would happen if the printer doesn't calibrate at all? Wouldn't work as normal without the bed mesh?

**Comment by Orel-A** [comment]
Are you asking what would happen if the user doesn't tick Calibrate and doesn't have the bed mesh gcode line that we've recently added? It'll load the previous bed mesh, which could mean an old 5x5 bed mesh.

Typically the user does this when he calibrates the printer for the first time, but he can go to the screen settings and there in Equipment Self-Check he has 2 options, Z-Offset calibration and Auto-Level calibration. If he selects both, the printer will do the z-offset first and proceeds to do Auto-Level correctly. However of he chooses to only do Auto-Level calibration, this won't heat the bed to 60C and will create problems on the next print run. Another problem is that a bed mesh heated to 60C isn't that accurate for 80C when printing Petg since the bed will warp differently... So a mix of various problems.

**Comment by cochcoder** [comment]
Ok, I'll leave as it is

**Comment by cochcoder** [comment]
> What should be the values for these parameters in the CR-10 SE?
> 
> orca-slicer_lfDHgz88bQ
> 
> 

It'll give a brief description of where to measure when you hover over the values. If anyone finds a dramatically different value for the stock hot end on these printers, let me know ;)

**Comment by Orel-A** [comment]
@cochcoder 
Hi, you accidentally changed the default line width to 0.45mm instead of the internal solid infill.
The bridge flow ratio of 0.95 is causing issues with PETG. Since the material likes to contract under cooling and finally lines snap as there is no enough material to hold the bridge. This happens on both eSun and PolyMaker. On PolyMaker specifically, to make a successful bridge one needs to enable Thick Bridges however. There is no apparent downside to this change on PLA.

Please make the following changes:
```
"line_width": "0.42"
"internal_solid_infill_line_width": "0.45"
"bridge_flow": "1"
"elefant_foot_compensation": "0.05"
"xy_hole_compensation": "0.025"
"outer_wall_acceleration": "4000"
```

**Comment by cochcoder** [comment]
> @cochcoder 
> 
> Hi, you accidentally changed the default line width to 0.45mm instead of the internal solid infill.
> 
> The bridge flow ratio of 0.95 is causing issues with PETG. Since the material likes to contract under cooling and finally lines snap as there is no enough material to hold the bridge. This happens on both eSun and PolyMaker. On PolyMaker specifically, to make a successful bridge one needs to enable Thick Bridges however. There is no apparent downside to this change on PLA.
> 
> 
> 
> Please make the following changes:
> 
> ```
> 
> "line_width": "0.42"
> 
> "internal_solid_infill_line_width": "0.45"
> 
> "bridge_flow": "1"
> 
> "elefant_foot_compensation": "0.05"
> 
> "xy_hole_compensation": "0.025"
> 
> "outer_wall_acceleration": "4000"
> 
> ```
> 
> 
> 
> EDIT: The change filament gcode should be M600 instead of PAUSE.

This is in regards to the Ender-3 V3 KE, right?

**Comment by Orel-A** [comment]
> This is in regards to the Ender-3 V3 KE, right?

Right.

**Comment by cochcoder** [comment]
When you enable thick bridges, how much is the quality affected? If there is a significant difference, then I think it should remain disabled.

**Comment by Orel-A** [comment]
I was more concerned that with eSun PETG the bridges were failing. Fortunately the bridge flow increase solves the issue for this filament.

**Comment by Orel-A** [comment]
Ah, I think I confused you. When I wrote "no apparent downside to this change on PLA" I didn't mean the thick bridge feature, I meant the bridge flow increase to 1 ratio.

**Comment by cochcoder** [comment]
> Ah, I think I confused you. When I wrote "no apparent downside to this change on PLA" I didn't mean the thick bridge feature, I meant the bridge flow increase to 1 ratio.

That would be it. 

Thank you for finding this changes, I've add them to #5743
